in_toto/verifylib.go

```diff
@@ -93,9 +93,9 @@ func verifyMatchRule(ruleData map[string]string,
 	var dstArtifacts map[string]interface{}
 	switch ruleData["dstType"] {
 	case "materials":
-		dstArtifacts = dstLinkMb.Signed.(Link).Materials
+		dstArtifacts = dstLinkEnv.GetPayload().(Link).Materials
```

Comment:
check type?

Reply:
We check to ensure it's not a layout prior to this in InTotoVerify. I guess a general question is if these checks need to happen everywhere for resilience?

Reply:
Mostly for resilience as it will cause a panic

Reply:
#230